### PR TITLE
[skip ci] Doc update about when to use quotes in vic-machine options

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic/issues/2073, which resulted from this [SocialCast posting](https://vmware-com.socialcast.com/messages/31466650).

Updates the following doc topics to describe when to wrap options in quotes:
- [Virtual Container Host Deployment Options](https://github.com/stuclem/vic/blob/master/doc/user_doc/vic_installation/vch_installer_options.md)
- [Virtual Container Host Inspect Options](https://github.com/stuclem/vic/blob/master/doc/user_doc/vic_admin/inspect_vch_options.md)
- [Virtual Container Host Delete Options](https://github.com/stuclem/vic/blob/master/doc/user_doc/vic_admin/delete_vch_options.md)
- [Virtual Container Host List Options](https://github.com/stuclem/vic/blob/master/doc/user_doc/vic_admin/list_vch_options.md)

Adds [VCH Deployment Fails with Unknown or Non-Specified Argument Error or Incorrect User Name Error](https://github.com/stuclem/vic/blob/master/doc/user_doc/vic_installation/ts_cli_argument_error.md) 

@emlin and @gigawhitlocks, please can you take a look?

Thanks!

### CODE REVIEW

File: doc/user_doc/vic_admin/delete_vch_options.md [hunk 1]
```diff
@@ -10,9 +10,13 @@ The IPv4 address, fully qualified domain name (FQDN), or URL of the ESXi host or
...
 - If the target ESXi host is managed by vCenter Server, or if you deployed the virtual container host to a cluster, provide the address of vCenter Server.<pre>--target <i>vcenter_server_address</i></pre>
-- You can optionally include the user name and password of the ESXi host or vCenter Server in the target URL. Wrap the user name or password in single quotes (Linux or Mac OS) or double quotes (Windows) if they include special characters.<pre>--target <i>username</i>:<i>password</i>@<i>esxi_or_vcenter_server_address</i></pre>
+- You can include the user name and password in the target URL. <pre>--target <i>vcenter_or_esxi_username</i>:<i>password</i>@<i>vcenter_or_esxi_address</i></pre>
+
+  Wrap the user name or password in single quotes (Linux or Mac OS) or double quotes (Windows) if they include special characters.<pre>'<i>vcenter_or_esxi_usern@me</i>':'<i>p@ssword</i>'@<i>vcenter_or_esxi_address</i></pre>
+  
+  If you do not include the user name in the target URL, you must specify the `user` option. If you do not specify the `passwd` option or include the password in the target URL, `vic-machine delete` prompts you to enter the password.
```
Comment (emlin) [inline doc/user_doc/vic_admin/delete_vch_options.md]: currently, the short name 'passwd' is changed back to full name 'password'.

Comment (stuclem) [inline doc/user_doc/vic_admin/delete_vch_options.md]: Is it going to change back again?

File: doc/user_doc/vic_admin/inspect_vch_options.md [hunk 2]
```diff
@@ -10,9 +10,12 @@ The IPv4 address, fully qualified domain name (FQDN), or URL of the ESXi host or
...
 - If the target ESXi host is managed by vCenter Server, or if you deployed the virtual container host to a cluster, provide the address of vCenter Server.<pre>--target <i>vcenter_server_address</i></pre>
-- You can optionally include the user name and password of the ESXi host or vCenter Server in the target URL. Wrap the user name or password in single quotes (Linux or Mac OS) or double quotes (Windows) if they include special characters.<pre>--target <i>username</i>:<i>password</i>@<i>esxi_or_vcenter_server_address</i></pre>
+- You can include the user name and password in the target URL. <pre>--target <i>vcenter_or_esxi_username</i>:<i>password</i>@<i>vcenter_or_esxi_address</i></pre>
+
+  Wrap the user name or password in single quotes (Linux or Mac OS) or double quotes (Windows) if they include special characters.<pre>'<i>vcenter_or_esxi_usern@me</i>':'<i>p@ssword</i>'@<i>vcenter_or_esxi_address</i></pre>
+  
+  If you do not include the user name in the target URL, you must specify the `user` option. If you do not specify the `passwd` option or include the password in the target URL, `vic-machine inspect` prompts you to enter the password.
```
Comment (emlin) [inline doc/user_doc/vic_admin/inspect_vch_options.md]: ditto

**Comment by emlin** [comment]
lgtm

**Comment by stuclem** [comment]
Thanks @emlin!

**Comment by andrewtchin** [comment]
lgtm

**Comment by stuclem** [comment]
Thanks @andrewtchin!

**Pull request merged**